lib/rails_response_dumper/option_parser.rb

```diff
@@ -18,6 +18,10 @@ def self.parse_options!
       opts.on('--order TYPE', 'Run dumps by the specified order type.') do |v|
         options[:order] = v
       end
+
+      opts.on('--exclude-response-headers', 'Do not output response headers') do |v|
```

Comment:
The other options all have punctuation.

```suggestion
      opts.on('--exclude-response-headers', 'Do not output response headers.') do |v|
```

Reply:
[Add punctuation](https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/68/commits/56d483769583c24ddd80e704311a690606a432ad)